examples/test.py

```diff
@@ -0,0 +1,71 @@
+import emod_api.campaign as campaign
+import numpy as np
+import pandas as pd
+from snt.hbhi.set_up_interventions import InterventionSuite
+from emod_api.interventions.common import BroadcastEvent, DelayedIntervention
+from emodpy_malaria.interventions.common import add_triggered_campaign_delay_event
+from snt.hbhi.set_up_interventions import add_all_interventions
+from emodpy_malaria.interventions.vaccine import add_scheduled_vaccine, add_triggered_vaccine
+import manifest
+
+def non_snt_camp():
+    import emod_api.campaign as campaign
+    campaign.schema_path = manifest.schema_file
+
+    delay_distribution = {"Delay_Period_Distribution": "CONSTANT_DISTRIBUTION",
+                          "Delay_Period_Constant": 274}
+    event_name = "RTSS_1_eligible"
+    broadcast_event = BroadcastEvent(campaign, event_name)
+    delayed_event = DelayedIntervention(campaign, Configs=[broadcast_event],
+                                        Delay_Dict=delay_distribution)
+    add_triggered_campaign_delay_event(campaign, start_day=1,
+                                       trigger_condition_list=['Births'],
+                                       demographic_coverage=0.504,
+                                       individual_intervention=delayed_event)
+    add_triggered_vaccine(campaign,
+                          start_day=1,
+                          trigger_condition_list=["RTSS_1_eligible"],
+                          intervention_name='RTSS',
+                          broadcast_event='Received_Vaccine',
+                          vaccine_type="AcquisitionBlocking",
+                          vaccine_initial_effect=0.8,
+                          vaccine_box_duration=0,
+                          vaccine_decay_time_constant=592.4067,
+                          efficacy_is_multiplicative=False)
+
+    return campaign
+
+
+def snt_camp():
+    import emod_api.campaign as campaign
+    campaign.schema_path = manifest.schema_file
+
+    int_suite =  InterventionSuite()
+    int_suite.rtss_auto_changeips = False
+
+    rtss_df = pd.DataFrame({
+        'DS_Name': ['Yomou'],
+        'coverage_levels': [0.504],
+        'round': [3],
+        'vaccine': ['simple'],
+        'initial_killing': [0.8],
+        'decay_time_constant': [592.4067],
+        'rtss_touchpoints': [274],
```

Comment:
I believe this needs to be 274.0 otherwise there's a type mismatch after it comes out of df. Or we need 
![image](https://github.com/InstituteforDiseaseModeling/emodpy-snt/assets/16845361/61260947-39a1-46c7-958e-df132f258760)

Reply:
the line needs to be uncommented. 
The error was the the touchpoints/triggers were read out as Int64 but that format didn't work with json, we needed them to be floats.